javav2/usecases/Creating_rds_item_tracker/Readme.md

```diff
@@ -48,6 +46,18 @@ To complete the tutorial, you need the following:
 + Running this code might result in charges to your AWS account. 
 + Be sure to terminate all of the resources you create while going through this tutorial to ensure that you’re not charged.
 
+### Creating the resources
+
+Create an Amazon RDS MySQL DB instance that has a table named **Work** and a primary key named **idwork** (this table is described in a following section). For information about creating an Amazon RDS database, see [Creating a MySQL DB instance and connecting to a database on a MySQL DB instance](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_GettingStarted.CreatingConnecting.MySQL.html). 
+
+After you create the database, in the **Connectivity & security** section, view the **Endpoint** and **Port** of the DB instance. You need these values when you create a connection to the database using the Java JDBC API (this is shown later in this tutorial). 
+
+![AWS Tracking Application](images/trackEndpoint2.png)
+      
+**Note:** You must set up an inbound rules for the security group to connect to the database from your development environment. Setting up an inbound rule essentially means enabling an IP address to use the database. Once you set up an inbound rule, you can connect to the database from a client such as MySQL Workbench. For more information, see [Controlling Access with Security Groups](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Overview.RDSSecurityGroups.html).  
```

Comment:
set up an inbound rules for --> set up an inbound rule for 
Once you set up --> After you set up